apps/api/drizzle/0061_famous_kate_bishop.sql

```diff
@@ -35,13 +35,14 @@ CREATE TABLE "seasons" (
 	CONSTRAINT "seasons_number_unique" UNIQUE("number"),
 	CONSTRAINT "seasons_name_unique" UNIQUE("name")
 );
+INSERT INTO seasons (id, number, name, start_date, end_date)
+	VALUES (0, 0, 'Genesis', '2025-10-13T00:00:00Z', null)
+	ON CONFLICT DO NOTHING;
+
```

Comment:
Ran the migration on a dump of prod and found this needs to happen before we add the foreign key to `conviction_claims`.